helix-term/src/commands.rs

```diff
@@ -2981,7 +2981,7 @@ pub mod insert {
                         for _ in 0..drop {
                             // delete up to `drop` spaces
                             match chars.next() {
-                                Some(' ') => start -= 1,
+                                Some(c) if c.is_ascii_whitespace() => start -= 1,
```

Comment:
I think this is an unnecessary change, as aside from line endings tab and space are the only two recognized as ascii whitespace.